changelogs/unreleased/clarity5-mklanjsek

```diff
@@ -0,0 +1 @@
+Upgraded project to Clarity 5
```

Comment:
Not blocking: Header spacing changes here (top -> 0.18, bottom -> current):

![image](https://user-images.githubusercontent.com/10288252/113032499-c4c54b80-9144-11eb-9dd8-ac59cfb7563e.png)